governance/approveDeny/README.md

```diff
@@ -31,15 +31,15 @@ Installation
    ```
 
 1. Download the following dependency jars, and copy them to the
-  `${ARTIFACTORY_HOME}/etc/plugins/lib` directory:
+  `${ARTIFACTORY_HOME}/var/etc/artifactory/plugins/lib` directory:
   * [HTTPBuilder](https://bintray.com/bintray/jcenter/org.codehaus.groovy.modules.http-builder%3Ahttp-builder/_latestVersion)
   * [Json-lib](https://bintray.com/bintray/jcenter/net.sf.json-lib%3Ajson-lib/_latestVersion)
   * [Xml-resolver](https://bintray.com/bintray/jcenter/xml-resolver%3Axml-resolver/_latestVersion)
 1. Edit the `approveDeny.json` file. See the example `approveDeny.json` provided or
    the details below on how to edit `approveDeny.json`.
-1. Copy `approveDeny.groovy` and `approveDeny.json` to the `${ARTIFACTORY_HOME}/etc/artifactory/plugins` directory.
+1. Copy `approveDeny.groovy` and `approveDeny.json` to the `${ARTIFACTORY_HOME}/var/etc/artifactory/artifactory/plugins` directory.
 1. There are two ways to load the plugin:
-   - Manually force the plugin to be loaded `curl -u username:password -XPOST -H'Content-Type: application/json' http://localhost:8080/artifactory/api/plugins/reload`
+   - Manually force the plugin to be loaded `curl -u username:password -XPOST -H'Content-Type: application/json' http://localhost:8082/artifactory/api/plugins/reload`
```

Comment:
I aways recommend referring to [netrc](https://everything.curl.dev/usingcurl/netrc) as it doesn't encourage people to put username and pass on the CLI:

`curl -sn ...`

Reply:
I'll update the doc to include netrc